alpa/device_mesh.py

```diff
@@ -83,13 +84,20 @@
 
 class DaemonMoveWorker:
     """A ray actor that moves local checkpoint into EFS in the background."""
-    def __init__(self, server_address: str):
-        print(server_address, "launch move worker successfully")
-    
-    def move(self, local_path, nfs_path):
-        pass
-
+    def move(self, from_dir: str, to_dir: str):
+        os.makedirs(to_dir, exist_ok=True)
+        process_pool = []
+        for file in os.listdir(from_dir):
+            from_path = os.path.join(from_dir, file)
+            to_path = os.path.join(to_dir, file)
+            process_pool.append(subprocess.Popen(["mv", from_path, to_path], 
+                                            stdin=None, stdout=None, stderr=None))
+        for p in process_pool:
```

Comment:
Does using multiprocessing here always make moving faster?

Reply:
No, the bottleneck is the EFS bandwidth. I will change it to `shutil.move`.